src/containers/AccessPointDetails/components/General/index.js

```diff
@@ -0,0 +1,1060 @@
+      bandwidth5U: data.details.radioMap.is5GHzU.channelBandwidth,
+      bandwidth5L: data.details.radioMap.is5GHzL.channelBandwidth,
+      perimeter2dot4: data.details.radioMap.is2dot4GHz.perimeterDetectionEnabled,
+      perimeter5U: data.details.radioMap.is5GHzU.perimeterDetectionEnabled,
+      perimeter5L: data.details.radioMap.is5GHzL.perimeterDetectionEnabled,
+      signal2dot4: data.details.radioMap.is2dot4GHz.neighbouringListApConfig.minSignal,
+      signal5U: data.details.radioMap.is5GHzU.neighbouringListApConfig.minSignal,
+      signal5L: data.details.radioMap.is5GHzL.neighbouringListApConfig.minSignal,
+      aps2dot4: data.details.radioMap.is2dot4GHz.neighbouringListApConfig.maxAps,
+      aps5U: data.details.radioMap.is5GHzU.neighbouringListApConfig.maxAps,
+      aps5L: data.details.radioMap.is5GHzL.neighbouringListApConfig.maxAps,
+      noiseFloor2dot4:
+        data.details.advancedRadioMap.is2dot4GHz.channelHopSettings.noiseFloorThresholdInDB,
+      noiseFloor5U:
+        data.details.advancedRadioMap.is5GHzU.channelHopSettings.noiseFloorThresholdInDB,
+      noiseFloor5L:
+        data.details.advancedRadioMap.is5GHzL.channelHopSettings.noiseFloorThresholdInDB,
+      noiseFloorTime2dot4:
+        data.details.advancedRadioMap.is2dot4GHz.channelHopSettings
+          .noiseFloorThresholdTimeInSeconds,
+      noiseFloorTime5U:
+        data.details.advancedRadioMap.is5GHzU.channelHopSettings.noiseFloorThresholdTimeInSeconds,
+      noiseFloorTime5L:
+        data.details.advancedRadioMap.is5GHzL.channelHopSettings.noiseFloorThresholdTimeInSeconds,
+      dfs2dot4: data.details.advancedRadioMap.is2dot4GHz.channelHopSettings.obssHopMode,
+      dfs5U: data.details.advancedRadioMap.is5GHzU.channelHopSettings.obssHopMode,
+      dfs5L: data.details.advancedRadioMap.is5GHzL.channelHopSettings.obssHopMode,
+      snr2dot4: data.details.advancedRadioMap.is2dot4GHz.bestApSettings.dropInSnrPercentage,
+      snr5U: data.details.advancedRadioMap.is5GHzU.bestApSettings.dropInSnrPercentage,
+      snr5L: data.details.advancedRadioMap.is5GHzL.bestApSettings.dropInSnrPercentage,
+      minLoad2dot4: data.details.advancedRadioMap.is2dot4GHz.bestApSettings.minLoadFactor,
+      minLoad5U: data.details.advancedRadioMap.is5GHzU.bestApSettings.minLoadFactor,
+      minLoad5L: data.details.advancedRadioMap.is5GHzL.bestApSettings.minLoadFactor,
+    });
+  }, []);
+
+  const columns = [
+    {
+      title: 'Wireless Network	',
+      dataIndex: 'name',
+      key: 'network',
+    },
+    {
+      title: 'SSID',
+      dataIndex: ['details', 'ssid'],
+      key: 'ssid',
+    },
+    {
+      title: 'Security Mode	',
+      dataIndex: ['details', 'secureMode'],
+      key: 'security',
+    },
+    {
+      title: 'Radio(s)',
+      dataIndex: ['details', 'appliedRadios'],
+      key: 'radios',
+      render: appliedRadios => appliedRadios.join(',  '),
+    },
+  ];
+
+  return (
+    <Form {...layout} form={form}>
```

Comment:
way too much duplicate code, reduce it by doing the renderItem logic as per my comment on Status.
Also per my comments on Status; radioMap and advancedRadioMap are dynamic and the inputs need to be able to handle any combo of radios per AP